addOns/automation/src/main/java/org/zaproxy/addon/automation/AutomationEnvironment.java

```diff
@@ -133,10 +133,38 @@ public Context parseContextData(
                 case "name":
                     name = replaceVars(value);
                     break;
+                case "urls":
+                    if (!(value instanceof ArrayList)) {
+                        progress.error(
+                                Constant.messages.getString(
+                                        "automation.error.context.badurlslist", value.toString()));
+
+                    } else {
+                        ArrayList<?> urlList = (ArrayList<?>) value;
+                        for (Object urlObj : urlList) {
+                            try {
+                                String url = replaceVars(urlObj);
+                                new URI(url, true);
+                                urls.add(url);
+                            } catch (URIException | NullPointerException e) {
```

Comment:
Can the `url` be `null`?

Reply:
I dont think urlObj can be and in any case replaceVars cant cope with a null input. The NullPointerException was added automatically by eclipse ;) Think it should be removed then?